src/fu-install-task.c

```diff
@@ -217,6 +232,20 @@ fu_install_task_check_requirements (FuInstallTask *self,
 		return FALSE;
 	}
 
+	/* check the version formats match */
+	fmt = fu_install_task_guess_version_format (self, version_release);
+	if (fmt != fu_device_get_version_format (self->device) &&
+	    (flags & FWUPD_INSTALL_FLAG_FORCE) == 0) {
```

Comment:
What do you think about in this circumstance that you did use force (to avoid the error) that the daemon would log that it overrrode the mismatching information to log?

Reply:
Just a g_warning?

Reply:
yeah